pkg/ingestor/parser/common/scanner/scanner.go

```diff
@@ -115,7 +118,64 @@ func purlsLicenseScanWithClient(
 }
 
 func PurlsDepsDevScan(ctx context.Context, purls []string) ([]assembler.CertifyScorecardIngest, []assembler.HasSourceAtIngest, error) {
-	return nil, nil, fmt.Errorf("Unimplemented")
+	certifyScorecards := []assembler.CertifyScorecardIngest{}
+	hasSourceAts := []assembler.HasSourceAtIngest{}
+
+	ddc, err := deps_dev_client.NewDepsClient(ctx)
```

Comment:
What is the added time to the ingestion when this is enabled? Can you post in a comment the time with this enabled and without?

Reply:
Added a comment here that each scan takes about 300ms - 500ms ranging from 1-30 PURLs.